force-app/main/default/classes/ChangeOnQuoteSentStageTest.cls

```diff
@@ -0,0 +1,40 @@
+@isTest
+private class ChangeOnQuoteSentStageTest {
```

Comment:
Every Test class is usually writing for one trigger/class. However testmethod is writing for some functionality done within this trigger. For this case it's better to have something like "OpportunityTriggerTest" for future extension. However you will have , let's say' "changeOnQuoteSentStageTest" method to test exactly this functionality